### ISSUE TITLE
Gerber file

### ISSUE THREAD

**Issue by KamenRiderV3** [particle-iot/thermostat#3]
Could you post the gerber files?  Thanks.

**Comment by technobly** [comment]
I don't believe there is a PCB layout, so no gerber files.  The whole thing was prototyped on a breadboard and then point to point wired on perfboard/vectorboard.  In fact, I don't see a schematic either, that would be a good first start :)

Since this is on github, I'm sure if a schematic is provided by Spark... someone could layout the pcb and submit a pull request ;-)

**Comment by mohitbhoite** [comment]
As @technobly said, the circuit was built on a proto board. We do have the schematics available though, I'll make a pull request today.

**Comment by KamenRiderV3** [comment]
Thanks!
